back-office/advanced-parameters/database/advanced_parameters-database-sql_manager.md

```diff
@@ -30,21 +30,84 @@ At first, the SQL Manager page displays 2 messages:
+After clicking on "Delete selected", a modal with title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462))
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the selected webservice keys are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+#### Columns
+
+The page displays the SQL Manager list with the following columns:
+
+1) ID
+
+It's possible to sort by id asc or desc and search by id.
 
+2) SQL query name
+
+It's possible to sort by SQL query name asc or desc and search by SQL query name.
+
+3) SQL query 
+
+It's possible to sort by SQL query asc or desc and search by SQL query.
+
+4) Actions 
+
+- View. Executes the query, and get the results displayed in an HTML table, right within the PrestaShop interface. This is the main action and it can be performed by clicking on the line.
 When the user views the SQL request with no result, a warning is displayed: "This SQL query has no result."
+- Export. Executes the query, and downloaded the result in CSV format.
 - Edit. The user can edit a query as often as necessary, to refine it and get better results.
 - Delete. Once a query is not used anymore (or simply because it does not work), you can delete it by clicking this button and confirming your choice.
+After clicking on _Delete_, a modal with title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" 
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the message is deleted and a successful message is displayed "Successful deletion."
+
+#### Multistore behavior
+
+As SQL query can't be associated to specific shop(s) during the creation, so all the SQL query from all stores are listed, no matter the context selected.
 
 ### Settings
 
-Select your default file encoding. You can configure the character encoding of the downloaded CSV file. The default, UTF-8, is recommended, but you can select ISO-8859-1 if need be.
+Select your default file encoding. You can configure the character encoding of the downloaded CSV file. The default, UTF-8, is recommended, but you can select ISO-8859-1 if need be through the drop-down.
+
+#### Multistore behavior
+
+In a specific shop context or in a group context, the "default file encoding" doesn't have checkbox and is disabled with an information message displayed when the user hovers the drop-down: "You can modify this parameter in all shops context only. Switch context to enable it." (see issue [#19356](https://github.com/PrestaShop/PrestaShop/issues/19356))
+
+In an all shops context, the the "default file encoding" doesn't have specific settings drop-down.
```

Comment:
```suggestion
In an all shops context, this feature is disabled. Merchants cannot select any file encoding.
```

Reply:
Humm, that's not the case, precisely, it can be changed only in all shop context.
The specific settings drop-down is one of the new components that will be integrated in 1.7.8 and allowing in all shops context to see for a parameter if it has a different value in a specific shop.
But as the file encoding has the same value for all shops, it will not be displayed

Reply:
Noted, thanks, I might have read the multistore specifications too quickly. ;-)